frame/ethereum/src/mock.rs

```diff
@@ -183,16 +182,8 @@ impl pallet_evm::Config for Test {
 	type WeightInfo = ();
 }
 
-parameter_types! {
-	pub const PostBlockAndTxnHashes: PostLogContent = PostLogContent::BlockAndTxnHashes;
-}
-
-impl Config for Test {
-	type RuntimeEvent = RuntimeEvent;
-	type StateRoot = IntermediateStateRoot<Self>;
-	type PostLogContent = PostBlockAndTxnHashes;
-	type ExtraDataLength = ConstU32<30>;
-}
+#[derive_impl(crate::config_preludes::TestDefaultConfig)]
+impl Config for Test {}
```

Comment:
Instead of fixing the `pallet-ethereum` tests directly, I modified the `pallet_evm::TestDefaultConfig` to set the minimum `FixedGasPrice` by default, which allows the `pallet-ethereum` tests to pass. However, I configured *a meaningful gas price* for its own mock. I would appreciate your feedback on this approach.